server/requirements_streamlit.txt

```diff
@@ -1,3 +1,3 @@
 requests==2.32.0
-streamlit==1.37.0
```

Comment:
why not ? there is a security warning coming with 1.35.0

Reply:
https://github.com/dscc-admin-ch/lomas/security/dependabot/20

Reply:
ah... but with 1.37 the streamlit app broke with the main page, [this one](https://discuss.streamlit.io/t/attributeerror-str-object-has-no-attribute-main-script-path/73585) so i revert back to 1.35.

I'll try with 1.39 that they released on the 01.10, maybe they fixed it

Reply:
oups, I didn't realize it has broken the page, it is better if it works indeed ^^

Reply:
the issue is actually coming from the st-pages package, updating it to 0.5.0 fixes the problem :)

Reply:
might be good to add tests that the streamlit app has no bug, I'll have a look

Reply:
nice!

Reply:
(Seems possible https://docs.streamlit.io/develop/api-reference/app-testing i made a ticket)

Reply:
good idea